src/driver/uniformly_random_test_point_generator.py

```diff
@@ -33,25 +33,31 @@
 class UniformlyRandomTestPointGenerator(TestPointGenerator):
     """Generates a collection of test points for a given simulation."""
 
-    def __init__(self, dataset: DataSet, rng: np.random.Generator):
+    def __init__(
+        self,
+        dataset: DataSet,
+        rng: np.random.Generator,
+        npoints: int = MINIMUM_NUMBER_OF_TEST_POINTS,
+    ):
         """Returns a UniformlyRandomTestPointGenerator.
 
         Args:
           dataset:  The DataSet for which test points are to be generated.
           rng:  A numpy Generator object that is used to seed the generation
             of random test points.
+          npoints: Number of points to generate.
         """
         super().__init__(dataset)
         self._rng = rng
+        self._npoints = npoints
 
     def test_points(self) -> Iterable[List[float]]:
         """Returns a generator for generating a list of test points.
 
         Returns:
           An iterable of spend vectors representing locations where
           the true reach surface is to be compared to the modeled reach
-          surface.  A minimum of MINIMUM_NUMBER_OF_TEST_POINTS will
-          be returned.
+          surface.
         """
```

Comment:
is it desirable to add `num_points = min(self._npoints, MAXIMUM_NUMBER_OF_TEST_POINTS)` just like in `LatinHypercubeRandomTestPointGenerator`?